README.md

```
@@ -268,25 +268,34 @@ pip install -e ".[dev]"
+##### Frontend (React) — eslint
+From `frontend` directory:
+**Action**: To lint and format your staged files, run the following command from the `
+```bash
+npm install
+npm run lint
+npm run lint:fix
+```
+
+##### Install & Run per-commit
 **Command:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)